assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+// import propTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import loadStylesheet from '../../../utils/loadStylesheet';
+import { GOOGLE_MENU_FONT_URL } from '../../../app/font';
+import {
+  GRID_ITEM_HEIGHT,
+  PANEL_WIDTH,
+  DropAnimation,
+  FadeInAnimation,
+  FlyInLeftAnimation,
+  FlyInRightAnimation,
+  FlyInTopAnimation,
+  FlyInBottomAnimation,
+  PulseAnimation,
+  RotateInLeftAnimation,
+  RotateInRightAnimation,
+} from './effectChooserElements';
+
+const Container = styled.div`
+  background: black;
+  width: ${PANEL_WIDTH}px;
+`;
+
+const GridItem = styled.button`
+  border: none;
+  background: #333;
+  border-radius: 4px;
+  height: ${GRID_ITEM_HEIGHT}px;
+  position: relative;
+  overflow: hidden;
+`;
+
+const Grid = styled.div`
+  display: grid;
+  justify-content: center;
+  gap: 15px 3px;
+  grid-template-columns: repeat(4, 58px);
+  padding: 15px;
+  position: relative;
+`;
+
+const GridItemFullRow = styled(GridItem)`
+  grid-column-start: span 4;
+`;
+
+const GridItemHalfRow = styled(GridItem)`
+  grid-column-start: span 2;
+`;
+
+const ContentWrapper = styled.span`
+  display: flex;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Teko', sans-serif;
+  font-size: 20px;
+  color: white;
+  text-transform: uppercase;
+`;
+
+export default function EffectChooser() {
+  useEffect(() => {
+    loadStylesheet(`${GOOGLE_MENU_FONT_URL}?family=Teko`).catch(function () {});
+  });
```

Comment:
Would using:
```
const {
    actions: { maybeEnqueueFontStyle },
  } = useFont();
```
prevent double loading this if one of the text sets or stories is already loading in Teko?

Or is this doing something that `maybeEnqueueFontStyle` doesn't do?

Reply:
`maybeEnqueueFontStyle` downloads only the chars we need for the font menu. Ideally we might make Teko a required app font.

Reply:
Ah makes sense. thanks for clarifying!